wallet/notifications.go

```diff
@@ -401,6 +452,33 @@ type Block struct {
 	Transactions []TransactionSummary
 }
 
+// TicketSummary contains the properties to describe a ticket's current status
+type TicketSummary struct {
+	Ticket  *TransactionSummary
+	Spender *TransactionSummary
+	Status  TicketStatus
+}
+
+// TicketStatus describes the current status a ticket can be observed to be.
+type TicketStatus int8
+
+const (
+	// TicketStatusLive any currently live ticket.
+	TicketStatusLive TicketStatus = iota
+	// TicketStatusUnmined any not yet mined ticket.
+	TicketStatusUnmined
```

Comment:
unmined and immature both come before the ticket being live, so those two should be values 0 and 1 respectively.